08.ls_object/file_list.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+class FileList
+  def initialize(params)
+    @params = params
+  end
+
+  def display_lists
+    file_match_option = @params[:a] ? File::FNM_DOTMATCH : 0
+    current_files = Dir.glob('*', file_match_option).map { |file| FileInfo.new(file) }
+    @params[:r] ? current_files.reverse! : current_files
+    if @params[:l]
+      total_block_number(current_files)
+      current_files.each { |file| display_file_details(file, current_files) }
```

Comment:
fileだとFileクラスっぽく見えるので、
```suggestion
      current_files.each { |file_info| display_file_details(file_info, current_files) }
```

みたいにしてFileInfoクラスのインスタンスであることが推測できるようにした方がベターです。